### PR TITLE
Add Breakfast Wrap

### PR DESCRIPTION
**Pull request by LinuxLounge22**

### CODE REVIEW

**Comment by Morelcia** [comment]
tags are missing

**Comment by LinuxLounge22** [comment]
I added the correct tags

File: src/breakfast-wrap.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Breakfast Wrap
+
+The basic parts of a full English in a wrap. Inspired by Macdonalds breakfast wraps.
```
Comment (hallzy) [inline src/breakfast-wrap.md:3]:
```suggestion
The basic parts of a full English in a wrap. Inspired by McDonald's breakfast wraps.
```

File: src/breakfast-wrap.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+- ⏲️ Prep time: 5 min
+- 🍳Cook time: 10 min
```
Comment (hallzy) [inline src/breakfast-wrap.md:6]:
```suggestion
- 🍳 Cook time: 10 min
```

**Comment by LinuxLounge22** [comment]
Committed suggested changes

File: src/breakfast-wrap.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- milk
+- ketchup (optional)
+
+## Directions
+
+1. Begin by cutting the sausages in half and frying them ensuring that they're cooked the whole way through.
+2. Next, fry the bacon.
+3. Next, possibly while the bacon is frying, crack an egg into a bowl, add a dash of milk and whisk it
+4. Then, put the egg mixture into the microwave for a minute, take it out and whisk it again, do this twice again or until the scrambled egg is ready.
+5. Add the sausage, bacon and scrambled egg to a wrap (I recommend two sausage halfs, a bacon rasher and half the bowl of scrambled egg per wrap). Add ketchup if desired.
```
Comment (hallzy) [inline src/breakfast-wrap.md:24]:
```suggestion
5. Add the sausage, bacon and scrambled egg to a wrap (I recommend two sausage halves, a bacon rasher and half the bowl of scrambled egg per wrap). Add ketchup if desired.
```

**Pull request merged**